### PR TITLE
Bring back typelib related attributes

### PR DESCRIPTION
**Pull request by yizhang82**
Not strictly required for net standard 2.0 but nice to have for tlbimp created dll to not have dangling references. 

@jkotas PTAL

### CODE REVIEW

**Comment by jkotas** [comment]
Why not to add them to corefx instead?

**Comment by yizhang82** [comment]
@jkotas I didn't consider it but that's a good point. It's unlikely we'll ever poke at them in the runtime. let me look into adding them into S.P.InteropServices. Thanks!

**Comment by hseok-oh** [comment]
@dotnet-bot test Ubuntu arm Cross Release Build

File: src/mscorlib/shared/System/Runtime/InteropServices/TypeLibAttributes.cs [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        internal TypeLibFuncFlags _val;
+        public TypeLibFuncAttribute(TypeLibFuncFlags flags)
+        {
+            _val = flags;
+        }
+        public TypeLibFuncAttribute(short flags)
+        {
+            _val = (TypeLibFuncFlags)flags;
+        }
+        public TypeLibFuncFlags Value { get {return _val;} }
```
Comment (danmosemsft) [inline src/mscorlib/shared/System/Runtime/InteropServices/TypeLibAttributes.cs]: tiny nit -- space before `return`

Comment (yizhang82) [inline src/mscorlib/shared/System/Runtime/InteropServices/TypeLibAttributes.cs]: Hmm.. desktop code was like this (this is a simply a copy/paste). Fixed.

**Comment by yizhang82** [comment]
I'll open a PR on corefx side.

**Pull request closed without merging**